core/src/main/java/nl/nn/adapterframework/unmanaged/DefaultIbisManager.java

```diff
@@ -240,7 +240,19 @@ public void handleAdapter(String action, String configurationName,
 			for (Configuration configuration : configurations) {
 				if (configuration.getRegisteredAdapter(adapterName) != null) {
 					Adapter adapter = configuration.getRegisteredAdapter(adapterName);
-					Receiver receiver = adapter.getReceiverByName(receiverName);
+
+					Receiver<?> receiver = adapter.getReceiverByName(receiverName);
+					RunStateEnum currentRunState = receiver.getRunState();
+					if (!currentRunState.equals(RunStateEnum.STARTED)) {
+						if (currentRunState.equals(RunStateEnum.STOPPING) || currentRunState.equals(RunStateEnum.STOPPED)) {
+							adapter.getMessageKeeper().info(receiver, "already in state [" + currentRunState + "]");
+							return;
+						} else {
+							adapter.getMessageKeeper().warn(receiver, "currently in state [" + currentRunState + "], ignoring start() command");
+							return;
+						}
+					}
```

Comment:
Code onder STOPRECEIVER lijkt heel erg op die onder STARTRECEIVER. Lijkt me goed om die te combineren in een method.